paddle/fluid/operators/elementwise/elementwise_op_function.h

```diff
@@ -258,14 +258,26 @@ class TransformFunctor {
 
   inline void RunRowWise(int n, int pre) const {
     platform::Transform<DeviceContext> trans;
-    trans(ctx_, x_, x_ + nx_, RowwiseTransformIterator<T, DeviceContext>(y_, n),
-          z_, func_);
+#ifdef PADDLE_WITH_MKLML
+#pragma omp parallel for
+#endif
+    for (int ni = 0; ni < pre; ni++) {
```

Comment:
Is this `for` slowers on `threads=1`?

Reply:
I suppose the cost is tiny, given in internal implementation, it uses std::transform which is a kind of loop actually. But for some cases, e.g. pre, n, post is not big enough, multi-theading may be slower than single due to thread spawn/converge cost.
Anyway, it is better to measure performance delta on single thread with this patch, @luotao1, do you have suggestion which workload can be used for measurement?

Reply:
I don't have inspired workload for measurement. Could you give some benchmark on it?
Or could you distinguish multi-thread and one-thread in codes?

Reply:
sure, I will give some benchmark for our typical workloads.